services/webnn/ort/tensor_impl_ort.cc

```diff
@@ -137,7 +136,6 @@ void TensorImplOrt::WriteTensorImpl(mojo_base::BigBuffer src_buffer) {
             CHECK_STATUS(GetOrtApi()->GetTensorMutableData(
                 buffer_state->GetExclusivelyLockedResource()->tensor(),
                 &ort_tensor_raw_data));
-            CHECK(ort_tensor_raw_data);
```

Comment:
Another question, what's the case when the `ort_tensor_raw_data` is null? Should we ensure the base::span().copy_from() is an no-op for this case?

Reply:
> We are not allowed to create a ml tensor with size 0 in which case the `ort_tensor_raw_data` is null:

Then I think it makes sense to keep this CHECK?

Reply:
Agree, done in the latest commit, PTAL, thanks!